src/main/java/the_warlord/powers/DoubleTeamPower.java

```diff
@@ -0,0 +1,72 @@
+package the_warlord.powers;
+
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardQueueItem;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+
+public class DoubleTeamPower extends CustomWarlordModPower {
```

Comment:
We'll probably want to remove this power at the end of turn, to be consistent with Blur, Amplify and Double Tap.